create_infotexts.py

```diff
@@ -30,14 +30,15 @@ def load_places_mapping():
     places_df = places_df.set_index("Nyckelord")
     places_df.columns = ["freq", "commonscat", "wikidata"]
 
-    places_df.replace("-", np.nan)
+    places_df[places_df == "-"] = None
```

Comment:
Reasonably so! I noticed that np.nan leaves the string "nan" in the fields which makes our boolean tests further down the line not to work! Not happy with Pandas due to this - it took me a long time to find out.